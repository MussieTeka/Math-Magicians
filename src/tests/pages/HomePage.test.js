import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../pages/HomePage';

describe('HomePage', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
