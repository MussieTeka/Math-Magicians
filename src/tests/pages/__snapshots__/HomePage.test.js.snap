// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage renders correctly 1`] = `
<div
  className="home-container"
>
  <h1>
    Welcome to Math Magicians!
  </h1>
  <p>
    Math can be a challenging subject, but with Math Magicians, we aim to make it easier for you.
  </p>
  <p>
    Our website offers a powerful online calculator that allows you to perform a wide range of calculations with ease. Whether you need to perform basic arithmetic or solve complex algebraic equations, our calculator has you covered. It's intuitive, user-friendly, and accurate, so you can be confident in your results.
  </p>
  <p>
    But we don't stop there. We believe that math is not just about solving problems, but also about discovering the beauty and creativity in numbers. That's why we've included a page dedicated to inspiring math quotes. These quotes come from some of the greatest mathematicians of all time, and they're sure to spark your curiosity and creativity.
  </p>
  <p>
    Whether you're a student looking for help with math homework, a professional needing quick calculations, or simply someone who loves the beauty of numbers, Math Magicians has something for you. So come and explore our website, and discover the magic of math!
  </p>
</div>
`;
